util/dvsim/doc/testplanner.md

```diff
@@ -0,0 +1,285 @@
+---
```

Comment:
Did you render this? I wish github could render md files in reviews.

Reply:
It can. In the Files tab, click on the "document" icon:

![image](https://user-images.githubusercontent.com/1467123/118009405-ee79b100-b345-11eb-97b2-b3d3172e688a.png)

Reply:
Thanks Philipp, yet some of the links are shown as
... The skeleton [DV document]({{< relref "dv_methodology/index.md#dv-document" >}}) and ...
instead of an actual link. Maybe it is a feature?

Reply:
Markdown has many dialects. GitHub only renders "github flavored markdown", while we're using "Hugo markdown". So some features, including the links, are not rendered properly in the GitHub preview.

If you'd like to see how the pages really look, you need to check out the pull request and run `./util/build_docs.py --preview` locally.

Reply:
Got it, thanks.